Source/Core/getAbsoluteUri.js

```diff
@@ -38,8 +38,13 @@ getAbsoluteUri._implementation = function (relative, base, documentObject) {
     base = defaultValue(documentObject.baseURI, documentObject.location.href);
   }
 
-  var baseUri = new Uri(base);
-  var relativeUri = new Uri(relative);
-  return relativeUri.resolve(baseUri).toString();
+  var relativeUri = new URI(relative);
+  if (relativeUri.is("urn")) {
+    return relativeUri.toString();
+  }
+  var url = relativeUri.absoluteTo(base).toString();
+  // URI.absoluteTo() escapes the placeholders. Undo that.
+  url = url.replace(/%7B/g, "{").replace(/%7D/g, "}");
```

Comment:
This code was copied from `Resource.prototype.getUrlComponent()`; it's needed because the `urijs` implementation of `absoluteTo` normalizes URLs (this is also consistent with the `URL(relative, base)` constructor in JS. Is this logic safe here?

Reply:
It almost feels like this fix is in the wrong place, like the calling code is the thing that should be handling `{` or perhaps a specialized function like `getAbsoluteTemplateStringUri.js`  Feels weird to just have something as generic as `getAbsoluteUri` embed code like this.

Reply:
I moved the string replace to the spec in question. Is there no risk of of this mattering at runtime because browsers treat encoded symbols the same as unencoded ones?

Reply:
I didn't realize the specs were the only place this happens.. did you verify any sandcastles using template uris for imager work?

Reply:
I verified that imagery Sandcastles work as expected, though there are bunch of `Failed to obtain image tile X ... Y: ... Level: ...` in the console.. this happens in the same way in `main`, doesn't seem like we have an open issue for it?

As far as I can tell only the spec calls `getAbsoluteUri` with a templated url - by the time the imagery urls make their way to `fetchImage` the template values are already string replaced correctly.